backend/src/index.js

```diff
@@ -2,17 +2,49 @@ import "dotenv/config";
 import cors from "cors";
 import express from "express";
 
+const uri = process.env.DATABASE_URI;
+//Please add database uri in your .env as follows
+
+// DATABASE_URI = mongodb+srv://astrocoders:astrocoders123@codemons.jscsc.mongodb.net/Codemons?retryWrites=true&w=majority;
+
```

Comment:
if they are not important please remove comments.

Reply:
done